apps/native/app/src/screens/applications/applications.tsx

```diff
@@ -139,13 +135,6 @@ export const ApplicationsScreen: NavigationFunctionComponent = ({
     [],
   )
 
-  useActiveTabItemPress(3, () => {
-    flatListRef.current?.scrollToOffset({
-      offset: -150,
-      animated: true,
```

Comment:
This was causing the screen to scroll a bit down. Removing at least for now, until we decide if we want this and then what pressing the active tab should do (was not doing anything I think).